package.json

```diff
@@ -64,7 +64,7 @@
     "format": "npx eslint \"**/*.js\" && npx prettier --check \"**/*.js\" \"**/*.ts\" \"**/*.md\" \"**/*.json\"",
     "format:fix": "npx eslint --fix \"**/*.js\" && npx prettier --write \"**/*.js\" \"**/*.ts\" \"**/*.md\" \"**/*.json\"",
     "lint": "node test/lint",
-    "fix": "npm run format:fix && node scripts/fix",
+    "fix": "npm run format:fix && node scripts/fix/index.js",
```

Comment:
Does this only apply to the fix script?  What about the lint script?

Reply:
[`stripExt(modulePath) === scriptPath`](https://github.com/tschaub/es-main/blob/8a569c0ca2345221e62a1dedeb00f9a29301178f/main.js#L23), so that one is okay.

Reply:
Ah right, thank you!